package.json

```diff
@@ -6,6 +6,7 @@
     "babel-preset-es2015": "6.18.0",
     "intl": "1.2.5",
     "isomorphic-fetch": "2.2.1",
+    "keymirror": "0.1.1",
```

Comment:
Nothing to do on this for now, but I feel like we should write our own version of this in some sort of utils.js instead of having to include a package just for a single, simple function.